core/domain/stats_domain_test.py

```diff
@@ -1113,6 +1113,35 @@ def test_validate_playthrough_id_type(self):
             'Expected each playthrough_id to be a string')):
             self.exp_issue.validate()
 
+    def test_comparison_between_exploration_issues_returns_correctly(self):
+        expected_customization_args = {
+            'time_spent_in_exp_in_msecs': {
+                'value': 0
+            },
+            'state_name': {
+                'value': ''
+            }
+        }
+        exp_issue1 = stats_domain.ExplorationIssue(
+            'EarlyQuit',
+            expected_customization_args,
+            [],
+            1,
+            True
+        )
+        exp_issue2 = stats_domain.ExplorationIssue(
+            'EarlyQuit',
+            expected_customization_args,
+            [],
+            2,
+            True
+        )
+        exp_issue3 = exp_issue1
```

Comment:
This is a reference assignment, there is no chance that this object will be different. To properly check the method create a new object but with new values.

Reply:
Updated.